cobc/ppparse.y

```diff
@@ -721,13 +724,55 @@ ppparse_clear_vars (const struct cb_define_struct *p)
 
 %%
 
+program_structure:
+  program_with_control_division
+| statement_list
+;
+
+/* GCOS 7 COBOL85 ref. manual p. 136: [...] If the replace-entry is present in
+the Substitution Section of the Control Division of a source program, that
+source program, including all contained programs, must contain no REPLACE
+statement.  Thankfully this helps avoiding some conflicts. */
+program_with_control_division:
+  control_division_prefix statement_list
+| control_division_no_replace statement_no_replace statement_list
+| control_division_no_replace
+| control_division_with_replace DOT statement_no_replace_list
+;
+
+control_division_no_replace:
+  control_division_prefix SUBSTITUTION_SECTION DOT
+;
+
+control_division_with_replace:
+ /* The second period could be optional. */
+  control_division_prefix SUBSTITUTION_SECTION DOT replace_statement
+;
+
+control_division_prefix:
+  CONTROL_DIVISION DOT
+  {
+	  if (!ppparse_verify (cb_control_division, _("Control division"))) {
```

Comment:
no need for a translation here, but all upper-case --> `ppparse_verify (cb_control_division, "CONTROL DIVISION")`

Reply:
As this cannot be part of any other construct just drop `YYERROR` here, which then should still have the possible error from cb_verify depending on the configuration, but still consume the tokens so not fail parsing later (which is currently seen in the test).

Reply:
Ok thanks for the hint!